test/tbbmalloc/test_malloc_overload.cpp

```diff
@@ -448,7 +448,11 @@ TEST_CASE("Main set of tests") {
     CheckMemalignFuncOverload(aligned_alloc, free);
 #endif
 
+#ifdef HAVE_MALLINFO2
+    struct mallinfo2 info = mallinfo2();
+#else
     struct mallinfo info = mallinfo();
+#endif
```

Comment:
The both function should be replaced and tested.

Reply:
I think we need to test `mallinfo` always and `mallinfo2` when availble, e.g.
```suggestion
#ifdef HAVE_MALLINFO2
    struct mallinfo2 info = mallinfo2();
#endif
    struct mallinfo info = mallinfo();
```